tests/aggregation/tests.py

```diff
@@ -1750,6 +1750,26 @@ def __init__(self, expr):
             ],
         )
 
+    def test_order_by_aggregate_default_alias(self):
+        publisher_books = (
+            Publisher.objects.values("book")
+            .annotate(Count("book"))
+            .order_by("book__count", "book")
```

Comment:
Having `book` instead of `book__id` in this order by clause was a bit unexpected for me. TIL that:

> If you try to order by a field that is a relation to another model, Django will use the default ordering on the related model, or order by the related model’s primary key if there is no Meta.ordering.

Thanks :trophy: 

Now, with this information, and considering that the following assertion assumes order by `book__id`, would you agree that this is strictly speaking more correct?

```suggestion
            .order_by("book__count", "book__id")
```

(In other words, if someone changes the `ordering` of Book, this test would be unaffected when using `book__id`.)

Reply:
That's a good call, I can change it to ensure there's no ambiguity regarding the expected ordering.